sdk/videoanalyzer/video-analyzer-edge/samples/typescript/sample.env

```diff
@@ -0,0 +1,13 @@
+# Used in most samples. Retrieve these values from a Cognitive Services instance
```

Comment:
I don't see any of these variables used in the sample?

Reply:
Yeah we don't need them for our sample. I have deleted the file